cars/v1/x_pack/README.md

```diff
@@ -1,18 +1,29 @@
-This directory contains example configurations for x-pack:
+# This directory contains example configurations for x-pack:
 
 * `security`: Configures TLS for all HTTP and transport communication using self-signed certificates.
-* `monitoring-local`: Enables x-pack monitoring with export to the current cluster.
+* `monitoring`: Enables x-pack monitoring with export to the current cluster.
 
 The configurations have been implemented so that you can either only one of them or both together, i.e. all of the following combinations will work:
 
-* `--elasticsearch-plugins="x-pack:security"`
-* `--elasticsearch-plugins="x-pack:monitoring-local"`
-* `--elasticsearch-plugins="x-pack:security+monitoring-local"`
+* `--car-plugins="x-pack-security"`
```

Comment:
* The command line parameter says `--car-plugins` but should be `--car`
* All of these are mixins so they require a "proper" car, i.e. it should be `--car="defaults,x-pack-security"`